docs/source/v2/custom-database-names.blade.md

```diff
@@ -7,7 +7,7 @@ section: content
 
 # Custom Database Names {#custom-database-names}
 
-To set the a database name for a tenant, use set `_tenancy_db_name` key in the tenant's storage.
+To use a specific database name for a tenant, set the `_tenancy_db_name` key in the tenant's storage.
```

Comment:
I'd still maybe go with `set a database name` over `use a database name`, because you're not just using it now, you're setting it for all future "tenancy sessions" .

I'm tired though, I'll make the decision in the morning.

Reply:
No worries. Feel free to edit it directly according to your decision!